### ISSUE TITLE
GPS location not being added to photo details.

### ISSUE THREAD

**Issue by RollyPeres** [jamesmontemagno/MediaPlugin#576]
## Bug Information

Version Number of Plugin: Latest stable
Device Tested On:
Simulator Tested On: Genymotion 2.12.1
Version of VS: 2017
Version of Xamarin:
Versions of other things you are using:

### Steps to reproduce the Behavior

Hi,

I apologize for not fully filling up the template. 
I am trying to use MediaPlugin to take a photo, attach some GPS details to it and then send it to a WebAPI as a byte[], api which will save this picture on a network drive. 
The picture is being saved but it has no GPS details attacked to it(as seen in the attached screenshot).
I've tried with both SaveMetaData true/false, same exact result as mentioned above.
Any ideas of what I'm doing wrong or is this a bug ? 
Thank you for you time!

### Expected Behavior
GPS details are added to the picture being saved to network drive.

### Actual Behavior
GPS details are missing.

### Code snippet

var picture = await CrossMedia.Current.TakePhotoAsync(new StoreCameraMediaOptions
            {
                DefaultCamera = CameraDevice.Rear,
                SaveToAlbum = true,
                Directory = "Photoshoots",
                Name = "test.jpg",
                Location = new Location
                {
                    Latitude = 52.0271723411551,
                    Longitude = -0.764563267810425
                },
                SaveMetaData = true
            });

using (var memoryStream = new MemoryStream())
            {
                await picture.GetStream().CopyToAsync(memoryStream);
                picture.Dispose();

// _taskService.SavePicture uses File.WriteAllBytes(networkPath, byteArray);
                await _taskService.SavePicture(memoryStream.ToArray(), "fromstream", "2222", TaskConstants.PictureSubfolderKind.Photoshoot);
            }

### Screenshots
![image_props](https://user-images.githubusercontent.com/29372054/42078254-765d3378-7b84-11e8-95c4-8e292bf20daa.jpg)

**Comment by jamesmontemagno** [comment]
This is a photo from the stream. You should just try to get the photo from the phone itself as you are most likely stripping the metadata off the file.

**Comment by RollyPeres** [comment]
I clearly did that. The photo is being saved on the device as well as being saved on the external file server. The problem is the GPS coordinates are not being shown in the picture's details - this is true for both the photo on the device and for the one on the file server. (when I right click the photo and go to details tab, no gps data).

**Comment by edamreed** [comment]
Using plugin version 4.0.1.1
From Android device, Google Pixel
If I copy the file to my PC, I can see the Lat and Long info in file properties.
Have you tried on an actual device, you only mention Simulator Tested On: Genymotion 2.12.1.

**Comment by edamreed** [comment]
Looks like the passed in location info will only be used if the picture does not have any location data in the exif.
Can you show the GPS details of the file properties, your screen shot only shows the image details.
![image](https://user-images.githubusercontent.com/5575429/42870007-8c7917d2-8a6e-11e8-89fa-6a74f0545a3c.png)

**Comment by RollyPeres** [comment]
It does show  image details only because it does not have any GPS details
as I already mentioned. The GPS tab in image properties does not appear at
all.

**Comment by edamreed** [comment]
@RollyPeres Ah OK, on my machine the GPS details were way down the bottom of the details tab.
Thought you might not have scroll down that far.

**Comment by RollyPeres** [comment]
Well, thank you for thinking so highly of me. Clearly I didn't take the
time to scroll down the details window but I did take the time to submit a
full detailed issue on github.
Anyways, thanks for your time.

**Comment by jamesmontemagno** [comment]
Can this be closed? Is there something in the docs @edamreed that we can add to clarify?

**Comment by edamreed** [comment]
@RollyPeres my apologies, just trying to help.
I guessed that because you included a screenshot to show that the GPS details were missing, and the screenshot is not of the section where the GPS details are displayed (on my files), that maybe you didn't know where the GPS details are actually displayed.
Sorry if I caused any offence.

Also, confused me when you said:-
"It does show  image details only because it does not have any GPS details"
I assumed you meant that the Image section is only displayed in the details tab because there are no GPS details.
As I'm sure you are aware, the Image section is separate to the GPS section and is displayed regardless of the presence of GPS details.

Again, sorry if I caused an offence.

I'll leave James to help you with this one.


@jamesmontemagno 
I assume that as we don't know why (without debugging it) the supplied location details are not saving, not much to add to the documentation?
